code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -278,6 +278,10 @@
 
 	return
 
+/mob/living/simple_animal/can_be_pulled_by(var/mob/pulling_mob)
+	if(locate(/obj/item/explosive/plastic) in contents)
+		to_chat(pulling_mob, SPAN_WARNING("You leave \the [src] alone. It's got live explosives on it!"))
+		return FALSE
```

Comment:
would you want to return parent (or just TRUE) if the animal doesn't have explosives on it?

Reply:
Yes